1-Draft/RFCNNNN-Simplify-Module-Manifests.md

```diff
@@ -0,0 +1,170 @@
+---
+RFC: RFCnnnn
+Author: Kirk Munro
+Status: Draft
+SupercededBy: 
+Version: 1.0
+Area: Module Manifests
+Comments Due: July 15, 2019
+Plan to implement: Yes
+---
+
+# Title
```

Comment:
```suggestion
# Introduction
```

Reply:
Whoops, that's what I get for filling out an RFC late at night when I'm tired. Fixed. Thanks.